test/IRGen/unmanaged_objc_throw_func.swift

```diff
@@ -0,0 +1,70 @@
+// RUN: %target-swift-frontend -emit-ir %s | %FileCheck %s
+
+import Foundation
+
+@objc protocol SR_9035_P {
+  func returnUnmanagedCFArray() throws -> Unmanaged<CFArray>
+}
+
+// CHECK: define hidden swiftcc %TSo10CFArrayRefa* @"$s25unmanaged_objc_throw_func9SR_9035_CC22returnUnmanagedCFArrays0G0VySo0H3RefaGyKF"(%T25unmanaged_objc_throw_func9SR_9035_CC* swiftself, %swift.error** noalias nocapture swifterror dereferenceable(8)) #0 {
```

Comment:
Using `CHECK-LABEL` instead of `CHECK` here helps FileCheck divide the file up into separate sections, which keeps it from exiting early when you're checking multiple things in one file.

Reply:
Also, if more tests get added to this file, that `#0` (the function attributes) might change. Since they're not important, it's probably best to use a regex match here: `#{{[0-9]+}}`.